packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -160,6 +161,7 @@ const RealAccountSignup = ({
                 />
             ),
             title: local_props => {
+                setIsAddOrManageAcc(true);
```

Comment:
This is not the correct line of code to set that flag as it is just to set the `title`.
can you set the modal height using `add-or-manage` classname, it already exist in the code.

Reply:
Let me look into it 🙏